docs/NNNN-hashed-n-tuple-storage-layout.md

```diff
@@ -0,0 +1,301 @@
+# OCFL Community Extension NNNN: Hashed N-tuple Storage Layout
+
+  * **Extension Name:** NNNN-hashed-n-tuple-storage-layout
+  * **Authors:** Peter Winckles
+  * **Minimum OCFL Version:** 1.0
+  * **OCFL Community Extensions Version:** 1.0
+  * **Obsoletes:** n/a
+  * **Obsoleted by:** n/a
+
+## Overview
+
+This storage root extension describes how to safely map OCFL object identifiers
+of any length, containing any characters to OCFL object root directories with
+the primary goals of ensuring portability and filesystem performance at the cost
+of directory name transparency.
+
+Using this extension, OCFL object identifiers are hashed and encoded as hex
+strings. These digests are then divided into _N_ n-tuple segments, which are
```

Comment:
Given the requirement that the hex-encoded strings be [lowercased](https://github.com/OCFL/extensions/pull/16/files#diff-d4391f68083fa05a302d6cf4291e830b4e0d66ed6459d8969ef73f5a1fe641aaR118), maybe it would useful to add the same language here as in @bcail 's PR:
https://github.com/OCFL/extensions/pull/19/files#diff-febde49a93addbd21a93eb858fa085161bc456d0acb38f4ef992e5c88f79d9e0R16

https://github.com/OCFL/extensions/pull/19/files#diff-febde49a93addbd21a93eb858fa085161bc456d0acb38f4ef992e5c88f79d9e0R16

Reply:
Personally, I do not think that parenthetical is helpful in the overview.

Reply:
As long as you considered it.

Now to verify the json-schema...

Reply:
I'll rewrite it as "encoded as lowercase hex strings"